webapp/src/Components/Player/Overview/SideBar/PlayerStatsCard.tsx

```diff
@@ -120,15 +123,27 @@ class PlayerStatsCardComponent extends React.PureComponent<Props, State> {
                                 <Grid item xs={12}>
                                     <List component="nav">
                                         {this.state.playerStats.playersInCommon.map((person) =>
-                                            <Link to={PLAYER_PAGE_LINK(person.id)} style={{textDecoration: "none"}}
-                                                  key={person.id}>
-                                                <ListItem button>
-                                                    <ListItemIcon>
-                                                        <Person/>
-                                                    </ListItemIcon>
-                                                    <ListItemText primary={person.name}/>
-                                                </ListItem>
-                                            </Link>
+                                            <>
+                                                <Link to={PLAYER_PAGE_LINK(person.id)} style={{textDecoration: "none"}}
+                                                      key={person.id}>
+                                                    <ListItem button>
+                                                        <ListItemIcon>
+                                                            {person.avatar ?
+                                                                <img alt={`${person.name}'s avatar`} height={"30px"}
+                                                                     src={person.avatar}/> : <Person/>}
+                                                        </ListItemIcon>
+                                                        {/*<ListItemText primary={person.name}/>*/}
+                                                        <Typography>{person.name}</Typography>
```

Comment:
Why remove the listitemtext?

Why not use a ListItemSecondaryAction for the button?
https://material-ui.com/components/lists/#list-controls

Reply:
tbh didn't know that existed. so that's why I removed it